mtools/test/test_mplotqueries.py

```diff
@@ -0,0 +1,35 @@
+import sys
+from datetime import datetime, timedelta
+from random import randrange
+
+from dateutil import parser
+from nose.plugins.skip import SkipTest
+from nose.tools import raises
+
+import os
```

Comment:
Review PEP style guide to sort imports https://www.python.org/dev/peps/pep-0008/#imports ;)